docs/source/distributions/building_distro.md

```diff
@@ -71,6 +71,8 @@ options:
                         found. (default: None)
   --print-deps-only     Print the dependencies for the stack only, without building the stack (default: False)
   --run                 Run the stack after building using the same image type, name, and other applicable arguments (default: False)
+  --export-dir EXPORT_DIR
+                        Export the build artifacts to a specified directory instead of building the container. This will create a tarball containing the Dockerfile and all necessary files to build the container. (default: None)
```

Comment:
nit: Maybe break up the overlong line into two and align, like for the `--config` option. However this is not consistent, maybe we should agree on a max. line length for help texts (like 80 or 120)